docs/faq.md

```diff
@@ -96,3 +96,31 @@ There are two possible approaches:
 
 1. Traverse schema (e.g. with json-schema-traverse) and replace every `$ref` with the referenced schema.
 2. Use a specially constructed JSON Schema with a [user-defined keyword](./keywords.md) to traverse and modify your schema.
+
+## Using Ajv in Internet Explorer 11
+
+Ajv v8 in Internet Explorer 11 (IE11) will not work straight out of the box. To use it you need to set the following options:
+
+```js
+const options = {
+  unicodeRegExp: false, // this enables 'pattern' RegExp
+  code: { es5: true },  // uses es5 specific code branches within Ajv
+};
+const ajv = new Ajv(options);
+```
+
+and transpile Ajv within the `node_modules` of your project.
```

Comment:
isn't it better to recompile from typescript? This way you'd have correct source maps (probably)... Just thinking...

Reply:
Also, I'd probably move a bit more condensed version "to using in browser" section instead of here - I think it's more likely to be found there...